### PR TITLE
Javascript capstone: Feature/hit api

### PR DESCRIPTION
**Pull request by Gnwin**
# Javascript Capstone Project

In this project, we used kanban board, webpack, apis to build a mini tv app.(Javascript capstone project)

according to this requirement:

https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md

### CODE REVIEW

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, Team👋,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🔴🔴



You have submitted a project, that has failed linters, Kindly fix them.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/assets/js/modules/__tests__/App.test.js [hunk 1]
```diff
@@ -0,0 +1 @@
+// test
```
Comment (Meltrust) [inline src/assets/js/modules/__tests__/App.test.js]: According to the requirements:

image

Therefore:

- [x] Please, include unit tests for all the counters, as required.

That way, we will meet this requirement 💪

File: src/assets/stylesheets/style.css [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+/* :root {
+  --border-bottom: 1px solid rgb(230, 230, 230);
+  --grey: rgb(246, 246, 246);
+  --flex: flex;
+  --border-radius: 10px;
+} */
+
+/* Mobile View */
+
+.html {
```
Comment (Meltrust) [inline src/assets/stylesheets/style.css:15]: - [ ] OPTIONAL: I highly suggest you align these "comment" buttons.  Your app will look a lot better:

image

File: src/assets/stylesheets/style.css [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+  font-weight: 600;
+  font-style: normal;
+} */
+
+/* :root {
+  --border-bottom: 1px solid rgb(230, 230, 230);
+  --grey: rgb(246, 246, 246);
+  --flex: flex;
+  --border-radius: 10px;
+} */
```
Comment (Meltrust) [inline src/assets/stylesheets/style.css]: - [x] Please, delete this old piece of commented code, in order to improve the cleanliness of the repo 💪

File: src/assets/stylesheets/style.css (hunk 2 above)
Comment (Meltrust) [inline src/assets/stylesheets/style.css:15]: - [ ]  OPTIONAL:  I can see you did a mobile view for your app! Congratulations.  I recommend fixing the header, so all the elements are accessible, or even hide the ones that are not used:

image

image

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @Gnwin,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To highlight:

- Very nice implementation! Looking great!✔️ 

You can do it

## Required Changes ♻️

- [ ] According to the requirements:\

image

Therefore, please make a video that meets all the requirements listed above.  Kindly remember, that the next reviewer will check if the video exists, and meets all the requirements.

Please, provide a link to this video.


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Cathella — APPROVED** [review]
## 🔥 Status: Approved 🟢

### Hi @Gnwin, @karam084 👋🏻, 

Good job making the requested changes 💪🏻! Your project looks amazing 🤩.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
rosycheeks-mochi-peach

**Pull request merged**